.rubocop.yml

```diff
@@ -4,5 +4,4 @@ AllCops:
   Exclude:
     - vendor/**/*
     - gemfiles/vendor/**/*
-
-
+    - spec/**/*
```

Comment:
Run `rubocop -a ; rubocop --auto-gen-config` and undo this pls.

Reply:
Sure thing!